default/stringtables/en.txt

```diff
@@ -5437,6 +5437,8 @@ Research
 METER_RESEARCH_VALUE_DESC
 '''The Research (or RP "Research Points") of all planets in the empire is combined to discover new [[encyclopedia ENC_TECH]].
 
+The Target Research is an extrapolated value of the Research output on a planet, making it fluctuate on several turns, according to different bonus/malus caused by some [[encyclopedia ENC_BUILDING_TYPE]]s built on the planet, some [[encyclopedia ENC_TECH]] researched by the empire, some [[encyclopedia ENC_SPECIAL]]s linked to the planet, and other various game effects (e.g. species traits).
```

Comment:
I don't like this description of Target meters. I'd rather say they are the stable output level the planet will approach given the current factors influencing that resource. If buildings, techs, or focus change, then the target level could shift.

Reply:
OK, thanks for the better wording. Let me know if it's correct:

`The Target Research is the stable output level a planet will approach, given the current factors influencing the Research output. Its value is connected to [[encyclopedia ENC_BUILDING_TYPE]]s built on the planet,  [[encyclopedia ENC_TECH]] researched by the empire,  [[encyclopedia ENC_SPECIAL]]s linked to the planet, and other various game effects (e.g. species traits or planetary focus). If these factors change, then the Target Research value is likely to shift.`

Reply:
The list of things that ends in "and other various" and then an example is awkward. Just include the example in the list, and / or "various other".